README.md

```diff
@@ -101,6 +101,17 @@ Here in the Quick Start, we give a brief introduction of using source code to in
 
 ## Build from source
 
+Skip this paragraph if you are using Windows. As we use Thrift for our RPC module (communication and
+protocol definition), we involve Thrift during the compilation, so Thrift compiler 0.13.0 (or
+higher) is required to generate Thrift Java code. Thrift officially provides binary compiler for
+Windows, but unfortunately, they do not provide that for Unix OSs. However, we compiled a Unix
+compiler ourselves and put it onto GitHub, and with the help of a maven plugin, it will be
```

Comment:
How about remove `put it onto GitHub`？ Users may mistakenly think that it is uploaded to git of our IoTDB repo？

Reply:
That does not matter wher we put it, and I only intend to tell about that the user must be able to access GitHub.